internal/docstore/mongodocstore/localmongo.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Go Cloud Development Kit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Runs only tests relevant to the current pull request.
+# At the moment, this only gates running the Wire test suite.
+# See https://github.com/google/go-cloud/issues/28 for solving the
+# general case.
+
+# https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
+set -euo pipefail
+
+echo "Starting MongoDB..."
+docker rm -f mongo 2> /dev/null || :
+docker run -d --name mongo  -p 27017:27017 mongo:4
+echo "...done"
```

Comment:
This output should probably include a line about how to terminate the container (`docker kill mongo` IIRC). (Here and elsewhere.)

Reply:
Why? You can stop/restart by running this script again. I don't necessarily expect people to need to stop them otherwise.

I guess I could add a "stop" arg to this script that would just stop everything if you think that's useful.

Reply:
Discussed offline. Rerunning the script would only restart. It would be nice to have the reminder that I have a Docker container to clean up later.

Reply:
Done, added an echo on how to clean it up. Example:

```
$ internal/docstore/mongodocstore/localmongo.sh 
Starting MongoDB...
mongo
...done. Run "docker rm -rf mongo" to clean up the container.
```